src/nvim/globals.h

```diff
@@ -847,8 +847,8 @@ EXTERN int* (*iconv_errno)(void);
 EXTERN int State INIT(= NORMAL);        /* This is the current state of the
                                          * command interpreter. */
 
-EXTERN int finish_op INIT(= FALSE);     /* TRUE while an operator is pending */
-EXTERN long opcount INIT(= 0);          /* count for pending operator */
+EXTERN bool finish_op INIT(= false);    // true while an operator is pending
+EXTERN int opcount INIT(= 0);           // count for pending operator
```

Comment:
this should be reverted to long as well (`opcount` is not even used in `ops.c`).